lib/i18n_data.rb

```diff
@@ -57,6 +57,8 @@ def fetch(type, language_code)
     def normal_to_region_code(normal)
       {
         "ZH" => "zh_CN",
+        "ZH-HK" => "zh_HK",
+        "ZH-TW" => "zh_TW",
```

Comment:
- this hash should be a constant
 - don't map these 1-off translations but use `.tr('-', '_')` so it works for everything
 - add a test

Reply:
Hi @grosser , thank you for your reply, I already add a test and update the PR,
but I'm not familiar with Travis CI, would you help how to pass the test ? Thank you.